vocode/streaming/client_backend/conversation.py

```diff
@@ -78,9 +80,11 @@ def get_conversation(
             agent=self.agent_thunk(),
             synthesizer=synthesizer,
             conversation_id=start_message.conversation_id,
-            events_manager=TranscriptEventManager(output_device, self.logger)
-            if start_message.subscribe_transcript
-            else None,
+            events_manager=(
+                TranscriptEventManager(output_device, self.logger)
+                if start_message.subscribe_transcript
```

Comment:
this seems like it could break existing flows. Are you sure it's necessary?

Reply:
this check also shouldn't be necessary, we already do it in `handle_websocket_call`

Reply:
I don't actually change anything here I think this is just the formatter

Reply:
The actual change is in the rest of the file